docs/development_suite/deploy/overview.md

```diff
@@ -104,7 +104,7 @@ It is important to **save** the configuration at least one time after the featur
 :::
 
 :::info
-In order to avoid unnecessary deployments, services created using an **advanced configuration** should be manually modified by **removing all the old interpolated annotations**.
+In order to avoid unnecessary deployments, services created using an **raw manifest configuration** should be manually modified by **removing all the old interpolated annotations**.
```

Comment:
```suggestion
In order to avoid unnecessary deployments, services created using a **raw manifest configuration** should be manually modified by **removing all the old interpolated annotations**.
```